pycode/memilio-epidata/memilio/epidata/getPopulationData.py

```diff
@@ -323,7 +327,7 @@ def get_population_data(read_data=dd.defaultDict['read_data'],
         merge_berlin=True, merge_eisenach=merge_eisenach, zfill=True))
     age_cols = df_pop_raw.loc[
         idCounty_idx[0]: idCounty_idx[1] - 2,
-        dd.EngEng['ageRKI']].copy().values
+        dd.EngEng['ageRKI']].values.copy()
```

Comment:
is the copy still necessary?

Reply:
no :)

Reply:
Sorry, it is indeed necessary to prevent that df_pop_raw is changed. Without the copy age_cols is read only.